addons/packages/ako-operator/1.5.0/README.md

```diff
@@ -1,22 +1,96 @@
-# ako-operator
+# Load Balancer Operator
 
-This package provides NSX Advanced Load Balancer using ako-operator.
+This package reconciles Cluster API objects and manages the lifecycles of load balancers using ako-operator.
 
 ## Components
 
-* ako-operator
+* [Deployment](./bundle/config/upstream/akooperator/deployment.yaml) for the Load Balancer Operator (ako-operator)
+* [CRD](./bundle/config/upstream/akooperator/akodeploymentconfig.yaml) for `AKODeploymentConfig`
+* [Two `AKODeploymentConfig` samples](./bundle/config/upstream/akooperator/akodeploymentconfig.yaml) for management and workload clusters respectively
+
+## Supported Providers
+
+The following tables shows the providers this package can work with.
+
+| AWS  |  Azure  | vSphere  | Docker |
+|:---:|:---:|:---:|:---:|
+|  ❌ |   ❌ | ✅  |  ❌  |
```

Comment:
Is there a plan to support other cloud providers?

Reply:
yes, multi cloud LB operator solution is in our roadmap.